src/main/scala/cromwell/engine/WorkflowDescriptor.scala

```diff
@@ -98,7 +98,7 @@ case class WorkflowDescriptor(id: WorkflowId,
     def copyFile(file: WdlFile): Try[Unit] = {
       val src = file.valueString.toPath(gcsFilesystem)
       val wfPath = wfContext.root.toPath(gcsFilesystem).toAbsolutePath // Absolute path of the workflow directory
-      val relativeFilePath = src.subpath(wfPath.getNameCount, src.getNameCount) // Path of the src file relatively to the workflow directory
+      val relativeFilePath = src.subpath(wfPath.getNameCount, src.getNameCount) // Path of the src file relative to the workflow directory
```

Comment:
It was helpful when trying to figure out how to construct the correct final path but it's not very useful in the final code yes. I'll remove them in my PR